documentation/reference/qfieldcloud/system.en.md

```diff
@@ -4,102 +4,107 @@ tx_slug: documentation_reference_qfieldcloud_system
+| Offline editing                 | cloud & cable | Vector       |
+| Directly access data source     | cloud & cable | Any          |
+| Remove from project             | cloud & cable | Any          |
+| Copy                            | cable         | Any          |
+| Keep existing (Copy if missing) | cable         | Vector       |
+
+
+#### Cloud action configuration
+
+The cloud action set in QFieldSync is applied by QFieldCloud only at the moment of packaging a project for QField.
+
+This is the behavior of QFieldCloud (`libqfieldsync`) with the layers:
+
+| Action                      | File based layer                                                                               | Service-based layer (e.g. WMS)    | Database layers (Postgres)                                                                       |
+|-----------------------------|------------------------------------------------------------------------------------------------|-----------------------------------|--------------------------------------------------------------------------------------------------|
+| Offline editing             | Create consolidated copy of the data on pull, apply delta file on push to original data source | N/A                               | Create a consolidated copy of the data on pull, apply delta file on push to original data source |
+| Directly access data source | Create a read-only copy in an individual GeoPackage                                            | No action on the layer            | No action on the layer                                                                           |
+| Remove from project         | Remove the layer from the project                                                              | Remove the layer from the project | Remove the layer from the project                                                                |
+
+In summary, with QFieldCloud:
+
+- *Offline editing* means that a offline copy of the data will be generated by QFieldCloud and downloaded to QField.
+The data from *Offline editing*-layers will all be stored as multiple layers within a single `data.gpkg`.
+Whenever a modification is made on QField, a JSON structure called a *Change* (or Delta) will be generated only for the features that have been modified, only for the attributes or the geometry that have been modified.
+On push, the original data source will be updated, including the database for database layers.
+- *Directly access data source* is mainly used for service-based layers that are located on a internet accessible server.
+The data in these layers are directly modified by QField.
+Examples are WFS, WM(T)S-layers or layers coming from a database such as PostGIS layers.
+If the layer is file-based it will be in read-only mode.
+- *Remove from project* will simply remove the layer from the project (not package it for QField).
+
+From QFieldSync it will be possible to update a project already loaded on QFieldCloud.
```

Comment:
With QFieldSync in QGIS it will be possible to update a project already loaded on QFieldCloud.

Reply:
```suggestion
With QFieldSync in QGIS it will be possible to update a project already loaded on QFieldCloud.
```